src/picongpu/submit/submitAction.sh

```diff
@@ -18,21 +18,34 @@
 # If not, see <http://www.gnu.org/licenses/>.
 #
 
+## check if input files are newer than the picongpu binary
+numNewerFiles=$(find $TBG_projectPath/include -type f \
```

Comment:
Could we pass the results of find to a variable and than using `wc`. This would allow to use the intermediate result in case of the error and reduce the file system.